apps/root/src/pages/transfer/components/recipient-address/components/add-contact-modal/index.tsx

```diff
@@ -0,0 +1,227 @@
+import React from 'react';
+import { FormattedMessage, defineMessage, useIntl } from 'react-intl';
+import { Button, Grid, Modal, SuccessCircleIcon, TextField, Typography, colors } from 'ui-library';
+import useContactListService from '@hooks/useContactListService';
+import useStoredContactList from '@hooks/useStoredContactList';
+import useValidateAddress, { ValidationOutput } from '@hooks/useValidateAddress';
+import CenteredLoadingIndicator from '@common/components/centered-loading-indicator';
+import styled from 'styled-components';
+import ErrorCircleIcon from 'ui-library/src/icons/errorCircle';
+
+interface AddContactModalProps {
+  open: boolean;
+  setOpen: React.Dispatch<React.SetStateAction<boolean>>;
+  goBackToTransfer: () => void;
+}
+
+const StyledStatusTitle = styled(Typography).attrs({ variant: 'h5' })`
+  ${({ theme: { palette } }) => `
+  font-weight: bold;
+  color: ${colors[palette.mode].typography.typo1};
```

Comment:
I'm starting to think that we should set this directly on the typography since its being used quite a lot everywhere

Reply:
Are you referring to color of h5? Or which style specifically

Reply:
yeah the colors on the variants of typography, it seems like we are adding the color each time we use them, and they seem to be quite repetitive

Reply:
There are two main cases where colors are already set up by yourself 😄 `body` & `bodySmall`. I've done a little research and didn't find any other common pattern relating Typography variant to color.
I have removed unnecessary color specifications in our codebase, as the variants already cover this for the two cases mentioned